### DISCUSSION TITLE
How to block a persistent client ?

### DISCUSSION THREAD

**Discussion by tigermick** [AdguardTeam/AdGuardHome#6778] [Q&A]
Hi,
Thank you for this ad and tracker blocker ! On my network, everything works fine ! 
But I have a question about the rules for blocking services on persistent clients ?
I add a persistent client with IPV4 identifier but if I just use IPV4 of this client, it can continue to access blocking services with its ipv6 address ...
I can add my persistent client's ipv6 and everything works fine, but that ipv6 address changes when client disconnected and reconnected later ....
How can I permanently block a service to this customer ? 
Is it possible to link IPV4 and IPV6 about the same client ?
Or another solution is possible to identify a client and block it without specify all its ip ?

**Comment by ainar-g** [comment]
> Is it possible to link IPV4 and IPV6 about the same client ?

Yes, you can add multiple identifiers for the same client using the :heavy_plus_sign: button.

>  Or another solution is possible to identify a client and block it without specify all its ip ?

See the [documentation on identifying clients](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients#identifying-clients).

**Comment by tigermick** [comment]
Hi @ainar-g,
Thank you to try to help me.
I saw I can add multiple identifiers for the same client using the ➕ button and I tried to add the device's ipv6 like other identifier but, later, after some halts and starts of my target device, its IPV6 changed ... And I must to add another IPV6 like identifier for this same device again.
With my router, I can set ipv4 with static lease  but not with ipv6 ....
